relate/settings.py

```diff
@@ -27,17 +29,22 @@
     assert _local_settings_file != os.environ["RELATE_LOCAL_TEST_SETTINGS"]
     _local_settings_file = os.environ["RELATE_LOCAL_TEST_SETTINGS"]
 
-local_settings = {
-        "__file__": _local_settings_file,
-        }
+if not os.path.isfile(_local_settings_file):
+    raise RuntimeError(
+        "Management command '%(cmd_name)s' failed to run "
+        "because '%(local_settings_file)s' is missing."
+        % {"cmd_name": sys.argv[1],
+           "local_settings_file": _local_settings_file})
+
+local_settings_module_name = (
+    os.path.splitext(os.path.split(_local_settings_file)[-1])[0])
```

Comment:
Assert that the file extension is `.py`?

Reply:
That's a good idea.